packages/flutter_tools/lib/src/web/bootstrap.dart

```diff
@@ -94,18 +94,35 @@ document.addEventListener('dart-app-ready', function (e) {
    styleSheet.parentNode.removeChild(styleSheet);
 });
 
+// Create a TrustedTypes policy so we can attach Scripts...
+var _ttPolicy;
+if (window.trustedTypes) {
+  _ttPolicy = trustedTypes.createPolicy("flutter-tools-bootstrap", {
+    createScriptURL: (url) => {
+      switch(url) {
+        case "mapper": return "$mapperUrl";
+        case "requireJs": return "$requireUrl";
+        default: console.error("Unknown Flutter Web bootstrap resource!", url);
+      }
+      return null;
+    }
+  });
+}
+
 // Attach source mapping.
+var mapperSrc = _ttPolicy ? _ttPolicy.createScriptURL("mapper"):"$mapperUrl";
```

Comment:
nit: surround the `:` with one space on each side

Reply:
Done in the new function